### PR TITLE
Ban domain that's spamming us [ci skip]

### PR DESCRIPTION
**Pull request by esoergel**
@dannyroberts

### CODE REVIEW

File: corehq/apps/api/resources/v0_4.py [hunk 1]
```diff
@@ -126,6 +126,9 @@ def xform_es(self, domain):
         return MOCK_XFORM_ES or XFormES(domain)
 
     def obj_get_list(self, bundle, domain, **kwargs):
+        if domain == 'hsph-betterbirth'
```
Comment (dimagimon) [inline corehq/apps/api/resources/v0_4.py]: E901 SyntaxError: invalid syntax

File: corehq/apps/api/resources/v0_4.py [hunk 2]
```diff
@@ -126,6 +126,9 @@ def xform_es(self, domain):
...
+            raise BadRequest("There have been too many requests from this domain."
```
Comment (dimagimon) [inline corehq/apps/api/resources/v0_4.py]: E113 unexpected indentation

**Comment by esoergel** [comment]
I opened up this ticket for discussion and followup: http://manage.dimagi.com/default.asp?237763

**Comment by benrudolph** [comment]
this is a pretty big client of ours i believe. we may want to ensure it's ok with @sheelio to do this

**Comment by benrudolph** [comment]
nvm ignore me, i see you already talked to sheel

**Pull request merged**